content/techniques/security.md

```diff
@@ -20,7 +20,23 @@ import * as helmet from 'helmet';
 app.use(helmet());
 ```
 
-> info **Hint** Note that `app.use(helmet())` must come before other calls to `app.use()` or setup functions that may call `app.use()`). This is due to the way the underlying platform (e.g., Express) works, where the order that middleware/routes are defined matters. If you use middleware like `helmet` or `cors` after you define a route, then that middleware will not apply to that route, it will only apply to middleware defined after the route.
+If you are using the `FastifyAdapter`, you'll need [fastify-helmet](https://github.com/fastify/fastify-helmet) instead:
+
+```bash
+$ npm i --save fastify-helmet
+```
+
+Once the installation is complete, you'll need to register it:
+
+```typescript
+import * as helmet from 'fastify-helmet';
+// somewhere in your initialization file
+app.register(helmet);
+// or, but not type safe
+app.getHttpAdapter().getInstance().register(helmet);
```

Comment:
```suggestion
// app.getHttpAdapter().getInstance().register(helmet);
```

I'm thinking that, even though you say "or the following", perhaps we want to ensure people aren't confused and use *both*.  Hence the suggestion to maybe comment this alternative form.  Thoughts?

Reply:
I agree with your suggestion. And having it as a comment is enough.